### ISSUE TITLE
how can I debug my configuration

### ISSUE THREAD

**Issue by predmond** [sdelements/lets-chat-ldap#1]
I've entered my ldap config in settings.yml and run 'npm start' to start lets-chat. I tried logging in but my credentials are rejected.

Is there log or something? Is there a way to test from the command-line?

**Comment by Finkregh** [comment]
a debug-flag would be really nice...

**Comment by easterism** [comment]
Yes, it would be great. Write down some working config at least. How should i type bindCredentials (login + password)&

**Comment by sibartlett** [comment]
I have updated the examples in the README - I hope they are more useful.

**Comment by memphisraynz** [comment]
Hi there,
Just wondering if you have had any success with LDAP yet?
I am in the same baot at the moment that I have configured it but I am not seeing anything in active directory so I am not sure of the issue.
Thanks

**Comment by Finkregh** [comment]
@memphisraynz you might want to try to debug the LDAP-network-stream via tcpdump/wireshark
